.github/component_owners.yml

```diff
@@ -23,14 +23,17 @@ components:
   src/OpenTelemetry.Extensions.AWS/:
     - srprash
     - atshaw43
+    - ppittle
   src/OpenTelemetry.Extensions.Enrichment/:
     - xakep139
   src/OpenTelemetry.Instrumentation.AWS/:
     - srprash
     - atshaw43
+    - ppittle
   src/OpenTelemetry.Instrumentation.AWSLambda/:
     - rypdal
```

Comment:
@rypdal @Oberon00 Please review as you're the current owners of AWS Lambda instrumentation.